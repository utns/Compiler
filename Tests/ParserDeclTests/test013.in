type
    a = 1..10;