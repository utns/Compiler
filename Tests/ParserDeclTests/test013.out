integer                  type 
float                    type 
char                     type 
a                  type alias  subrange 1..10
